rehttp.go

```diff
@@ -264,7 +269,24 @@ func ExpJitterDelay(base, max time.Duration) DelayFn {
 		exp := math.Pow(2, float64(attempt.Index))
 		top := float64(base) * exp
 		return time.Duration(
-			PRNG.Int63n(int64(math.Min(float64(max), top))),
+			rand.Int63n(int64(math.Min(float64(max), top))),
+		)
+	}
```

Comment:
Let's have this function call `ExpJitterDelayWithRand` and pass `rand.Int63n` as the generator function, so that the algorithm is implemented only in one place.